src/App.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] As mentioned by the previous reviewers, please do not disable linters. They are tools that analyze your code to identify and flag potential issues, errors, or stylistic inconsistencies. They are widely used in software development to ensure code quality and maintainability. Therefore I suggest fixing the errors instead of disabling them